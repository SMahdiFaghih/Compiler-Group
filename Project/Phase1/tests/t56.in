2       3       ab2\t b