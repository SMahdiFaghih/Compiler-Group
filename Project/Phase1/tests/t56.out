T_INTLITERAL 2
T_INTLITERAL 3
T_ID ab2
I_ID b